tests/test_API_FB.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from API_FB import Manipulation
+
+class main():
+    # Load our first page
+    search_url = "https://www.facebook.com/careers/jobs/?q="
+
+    # Need to pull from db
+    prefrences = "DevOps"
+    r = requests.get(search_url+prefrences)
+
+    # Checking if content is available
+    # print(r.status_code)
+
+    soup = bs(r.content, 'html.parser')
+    # print(soup.prettify())
+
+    Manipulation.extract_jobs_list(soup)
+
+    Manipulation.extract_jobs_title(soup)
+
+    # Running the tests
+    pytest
```

Comment:
when you call pytest from your CI, you generally don't call pytest from within the file. Your functions should all have the word test_ as a prefix (such as def test_verify_jobs_list, ...) and the CI will call pytest on all files that match the python_files strigns in your pytest.ini